### DISCUSSION TITLE
Uncaught Error  at mutationReducer

### DISCUSSION THREAD

**Discussion by javiermelendezc** [TanStack/query#552] [General]
I'm new to react-query, and I've been going through all the examples.
Once I try to implement a useMutation on one of my projects it always throw **_Error at mutationReducer_**, and I couldn't find any issue regarding this. 
I even removed the call to mutateAddTodo, but it still throw this error. Am I missing some configuration Context or something?

```
function Feedback(props) {
    const [mutateAddTodo] = useMutation(
        value => fetch(`/api/data?add=${value}`),
        {
          onSuccess: () => queryCache.refetchQueries('todos'),
        }
      )
  return <div>Trying mutation</div>
}
```

Has any of you gone through this?

**Comment by wolverineks** [comment]
could you provide an example on codesandbox.io?
a good place to start
https://codesandbox.io/s/react-query-starter-8i7vb?file=/src/App.tsx

**Reply by javiermelendezc** [reply]
Hey @wolverineks the problem was my react version
I was  in 16.8.0-alpha.1, updated to 16.13.1 and everything worked
Thanks!
